src/test/java/com/iexec/worker/config/SchedulerConfigurationTests.java

```diff
@@ -28,8 +28,11 @@ class SchedulerConfigurationTests {
 
     @Test
     void shouldCreateBeanInstance() {
-        runner.withPropertyValues("core.protocol=http", "core.host=localhost", "core.port=13000")
+        runner.withPropertyValues("core.protocol=http", "core.host=localhost", "core.port=13000", "core.poolAddress=0x365E7BABAa85eC61Dffe5b520763062e6C29dA27")
                 .withConfiguration(UserConfigurations.of(SchedulerConfiguration.class))
-                .run(context -> assertThat(context).hasSingleBean(SchedulerClient.class));
+                .run(context -> {
+                    assertThat(context).hasSingleBean(SchedulerClient.class);
+                    assertThat(((SchedulerConfiguration) context.getBean("schedulerConfiguration")).getWorkerPoolAddress()).isEqualTo("0x365E7BABAa85eC61Dffe5b520763062e6C29dA27");
```

Comment:
Just wanted to see if there was an alternative to the proposed cast
```suggestion
                    assertThat(context).getBean("schedulerConfiguration", SchedulerConfiguration.class)
                            .extracting("poolAddress")
                            .isEqualTo("0x365E7BABAa85eC61Dffe5b520763062e6C29dA27");
```

Reply:
I wanted to try a full comparison with a `SchedulerConfiguration` but Spring enhances the class with new fields and the context bean is difficult to compare with a `SchedulerConfiguration` instance.
It is difficult to check `isEqualTo(expectedConfiguration)`.